subsys/net/ip/net_context.c

```diff
@@ -1532,6 +1532,44 @@ static void context_finalize_packet(struct net_context *context,
 	}
 }
 
+static struct net_pkt *context_alloc_pkt(struct net_context *context,
+					 size_t len)
+{
+	struct net_pkt *pkt;
+
+#if defined(CONFIG_NET_CONTEXT_NET_PKT_POOL)
+	if (context->tx_slab) {
+		pkt = net_pkt_alloc_from_slab(context->tx_slab(),
+					      PKT_WAIT_TIME);
```

Comment:
I propose that the timeout is placed as a parameter to context_alloc_pkt() so it can be modified in one place.

Reply:
ok